neptune_tensorflow_keras/impl/__init__.py

```diff
@@ -137,19 +137,23 @@ def _log_metrics(self, logs, category: str, trigger: str):
                 pass
 
     def on_train_begin(self, logs=None):  # pylint:disable=unused-argument
-        self._model_logger["summary"] = _model_summary_file(self.model)
         self._model_logger["optimizer_config"] = self.model.optimizer.get_config()  # it is a dict
         self._metric_logger["fit_params"] = self.params
 
         if self._log_model_diagram:
```

Comment:
I think we should also move `_log_model_diagram` to `on_train_end` (I think this might fail as well 🤔 )

Reply:
You are correct, fixed.